man/ocitools-generate.1.md

```diff
@@ -137,18 +137,21 @@ read the configuration from `config.json`.
   The special *PATH* `host` removes any existing PID namespace from
   the configuration.
 
-**--poststart**=CMD
-  Path to command to run in poststart hooks. This command will be run before
-  the container process gets launched but after the container environment and
-  main process has been created.
-
-**--poststop**=CMD
-  Path to command to run in poststop hooks. The command will be run
-  after the container process is stopped.
-
-**--prestart**=CMD
-  Path to command to run in prestart hooks. This command will be run before
-  the container process gets launched but after the container environment.
+**--poststart**=CMD[:ARGS..]
+  Set command to run in poststart hooks. Can be specified multiple times.
+  The multiple commands will be run in order before the container process
+  gets launched but after the container environment and main process has been
+  created.
+
+**--poststop**=CMD[:ARGS...]
+  Set command to run in poststop hooks. Can be specified multiple times.
+  The multiple commands will be run in order after the container process
+  is stopped.
+
+**--prestart**=CMD[:ARGS...]
+  Set commands to run in prestart hooks. Can be specified multiple times.
+  The multiple commands will be run in order before the container process
+  gets launched but after the container environment.
```

Comment:
On Wed, Aug 17, 2016 at 06:10:00PM -0700, Ma Shimiao wrote:
> On 08/18/2016 12:27 AM, W. Trevor King wrote:
> > Although, again, with opencontainers/runtime-spec#483
> > <https://github.com/opencontainers/runtime-spec/pull/483>
> > presumably landing soon it may not be worth spending time
> > polishing this up.
> 
> I'm not sure when #483 will be merged.

Sounds like “never” [1], so keep at it ;).

[1]: https://github.com/opencontainers/runtime-spec/pull/483#issuecomment-240568422